src/main/resources/config/liquibase/changelog/20231016123944_changelog.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd">
+    <changeSet author="valentinboehm" id="20231016123944">
+        <addColumn tableName="student_exam">
+            <column name="abandoned" type="boolean" valueBoolean="false">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
```

Comment:
Note for your future PRs: Since there is a clear default value for this attribute both logically and in the Java code, you could also specify it here. Would also prevent the DB from breaking exams on other PRs 😉